src/browser-realm.ts

```diff
@@ -19,6 +19,40 @@ const rawEventTargetProto = ReflectGetPrototypeOf(rawWindowPropertiesProto);
 export default function createSecureEnvironment(distortionMap?: Map<SecureProxyTarget, SecureProxyTarget>): (sourceText: string) => void {
     // @ts-ignore document global ref - in browsers
     const iframe = document.createElement('iframe');
+
+    // A comprehensive list of policy feature directives can be found at
+    // https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Feature-Policy#Directives
+    // Directives not currently supported by Chrome are commented out because
+    // Chrome logs warnings to the developer console.
+    iframe.setAttribute('allow', 
+        "accelerometer 'none';" +
+        "ambient-light-sensor 'none';" +
+        "autoplay 'none';" +
+        // "battery 'none';" +
+        "camera 'none';" +
+        // "display-capture 'none';" +
+        "document-domain 'none';" +
+        "encrypted-media 'none';" +
+        // "execution-while-not-rendered 'none';" +
+        // "execution-while-out-of-viewport 'none';" +
+        "fullscreen 'none';" +
+        "geolocation 'none';" +
+        "gyroscope 'none';" +
+        // "layout-animations 'none';" +
+        // "legacy-image-formats 'none';" +
+        "magnetometer 'none';" +
+        "microphone 'none';" +
+        "midi 'none';" +
+        // "navigation-override 'none';" +
+        // "oversized-images 'none';" +
+        "payment 'none';" +
+        "picture-in-picture 'none';" +
+        // "publickey-credentials 'none';" +
+        "sync-xhr 'none';" +
+        "usb 'none';" +
+        // "wake-lock 'none';" +
+        "xr-spatial-tracking 'none';"
+    );
     iframe.setAttribute('sandbox', 'allow-same-origin allow-scripts');
```

Comment:
probably the same question about `sandbox` attribute. These were things we were exploring before the detaching capability was found, so I wonder.